### PR TITLE
Public visit pages formatting and content

### PR DESCRIPTION
**Pull request by chrishanes**
The formatting of the visit pages look odd - especially the information around the confirmation date and visitors etc. The content also needs tidying up.

### CODE REVIEW

File: spec/features/maintaining_a_visit_spec.rb [hunk 1]
```diff
@@ -35,10 +39,10 @@
     cassette_name: :maintianing_a_visit_viewing_rejected
   } do
     visit visit_path(id: 'FOOBAR', locale: 'en')
-    expect(page).to have_text('Your visit request cannot take place')
+    expect(page).to have_text('Your visit request has been rejected')
 
-    click_link 'new visit'
-    expect(page).to have_css('h1', text: 'Who are you visiting?')
+    # click_link 'new visit'
```
Comment (TobyRet) [inline spec/features/maintaining_a_visit_spec.rb]: @KPobeeNorris - should this be commented out?

Comment (TobyRet) [inline spec/features/maintaining_a_visit_spec.rb]: @KPobeeNorris - should these lines be commented out

Comment (KPobeeNorris) [reply spec/features/maintaining_a_visit_spec.rb]: @TobyRet - As far as I can see this was commented out as part of @chrishanes's work yesterday. @chrishanes I tested it again with these lines back in and the test passes; are you ok with them being added in?

Comment (chrishanes) [reply spec/features/maintaining_a_visit_spec.rb]: @TobyRet @KPobeeNorris Sorry, yes those were supposed to be uncommented.

**Pull request merged**